2-virtual-machines-containers/2.1-golden-images.md

```diff
@@ -5,14 +5,30 @@
 ![](img2/linux_light.svg ':size=100x100 :class=light-mode-icon')
 ![](img2/linux_dark.svg ':size=100x100 :class=dark-mode-icon')
 
+?> If you have an M1 Mac checkout VirtualBox beta build for arm64. ( Note that as of 23/02/2023 the VirtualBox beta build for arm64 did not work)
+
+?> assuming you have an m1 mac & the virtualbox beta is still not operational, the only m1 compatible section is the compatibility section.
+
+### M1 Compatibility Excersize
+
+1. You will need to install the [UTM](https://mac.getutm.app) project.
+
+?> Assuming you have brew  `brew install --cask utm` should work
+
+2. You will also need an ARM64 linux iso; ideally we want to use the [CentOS Stream 9](https://www.centos.org/download/) iso located  make sure to select the *ARM64* version.
+
+3. Using UTM, create a new virtual machine and install CentOS using the ISO you downloaded.
```

Comment:
Inconsistent casing with `iso`. I I have seen it capitalized in other parts of the boot camp like [here](https://devops-bootcamp.liatr.io/#/2-virtual-machines-containers/2.1-golden-images?id=exercise-1-manual-build). But above in step 2 above you have it lower case. I don't really care either way I would just like us to be consistent.

Reply:
https://github.com/liatrio/devops-bootcamp/pull/232/commits/5e2b6d218128c803025422540e06775295e52bc6